docs/usage/faq.md

```diff
@@ -469,3 +469,19 @@ pyodide.runPython(`
     print(js_lambda, js_lambda_, js_lambda__) # 7, 7, 8
 `);
 ```
+
+## Can I use multi-threading/processing?
+
+Due to the limitation of current [wasm](https://pyodide.org/en/stable/usage/wasm-constraints.html)
+all packages that use parallelism (eg. rely on _fork_ and _pthread_)
+**may not work**. Accepting the performance degradation, a workaround
+consists of locking the number of threads down to `1` like shown below:
```

Comment:
```suggestion
No, fork and pthreads do not work in Pyodide. Attempts to use `threading`, `multiprocessing`, or `subprocess` will raise an `OSError`. You may be able to work around this by setting the number of threads to 1:
```

Reply:
Should we also mention that any attempts to use their APIs will cause fatal errors but simply `import threading`, etc. should not cause problems?

Reply:
> Should we also mention that any attempts to use their APIs will cause fatal errors but simply `import threading`, etc. should not cause problems?

You're right. General info APIs and synchronous  work on the main thread, but I don't know how reliable the results are. For example this works:

```pycon
>>> current = threading.current_thread()
>>> current.name
'MainThread'
>>> current.is_alive()
True
>>> import threading
>>> def target (nums):
...     print(sum(nums))
... 
>>> t = threading.Thread(target=target, args = ([1,2,3],))
>>> t.run()
6
>>>
```

Reply:
> any attempts to use their APIs will cause fatal errors

I don't believe that anything causes fatal errors. Just normal Python `OSError`s.

Reply:
Yes

Reply:
Maybe it's still better to guard the imports (or bring them into the functions where they are used) anyway.